packages/services/api/src/modules/shared/resolvers/JSON.ts

```diff
@@ -0,0 +1,7 @@
+import { JSONResolver } from 'graphql-scalars';
+
+// `scalar JSON` in `module.graphql.ts` does not have a description
+// and it messes up the static analysis
+JSONResolver.description = undefined;
```

Comment:
This comment only seems to be applicable to `JSONResolver` so I only left it here, and not on other scalar resolvers e.g. `DateTime`, `Date`, etc.

I'm happy to repeat the comment in other places it required 🙂